microsoft-edge/progressive-web-apps-chromium/whats-new/PWAs.md

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
Rename the file to all lowercase, such as "pwas.md" or "pwa.md".  The repo TOC doesn't list an existing file named "pwa.md".  The only files with uppercase are special top-level files like CONTRIBUTING.md.  Lowercase is probably more robust, and will work regardless of server changes.  Uppercase could break in future.